### ISSUE TITLE
New prebuilds in arcade

### ISSUE THREAD

**Issue by nohwnd** [dotnet/source-build#4188]
### Describe the Bug

After updating arcade I am seeing new prebuilts, which are not in baseline. 

https://github.com/microsoft/vstest/pull/4915/checks?check_run_id=22383316793

.packages/microsoft.dotnet.arcade.sdk/9.0.0-beta.24156.2/tools/SourceBuild/AfterSourceBuild.proj#L80
.packages/microsoft.dotnet.arcade.sdk/9.0.0-beta.24156.2/tools/SourceBuild/AfterSourceBuild.proj(80,5): error : (NETCORE_ENGINEERING_TELEMETRY=AfterSourceBuild) 3 new pre-builts discovered! Detailed usage report can be found at /__w/1/s/artifacts/sb/prebuilt-report/baseline-comparison.xml.
See https://aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them.
Package IDs are:
Microsoft.AspNetCore.App.Ref.8.0.1
Microsoft.NETCore.App.Host.linux-x64.8.0.1
Microsoft.NETCore.App.Ref.8.0.1

Will those be added to SB baseline, or I should do something else to eliminate them?

**Comment by MichaelSimons** [comment]
These prebuilts will need to be addressed - adding them to the baseline would not be appropriate.  I'm thinking these are coming from a project targeting net8.0 instead of netcurrent/net9.0.  The SDK update is what brings in 8.0.1 vs the previous 8.0.0 ref pack which is in SBRP.

**Comment by nohwnd** [comment]
Is this possibly caused by this https://github.com/microsoft/vstest/pull/4856/files (Update sourcebuild configuration to build net previous and net current) ? Do you see a way we can unblock ourselves?

**Comment by MichaelSimons** [comment]
Yes, that looks suspect.  In the source-build binlog, that is net8.0.  Wondering if that line can be conditioned out for source-build or set to NetCurrent for source-build?

**Comment by nohwnd** [comment]
I just noticed that there are already exclusions for 8.0.0 in eng/SourceBuildPrebuiltBaseline.xml in that PR, and arcade brings 8.0.1. Is that okay to update those exclusions, or that should not have been done in the first place?

**Comment by MichaelSimons** [comment]
> I just noticed that there are already exclusions for 8.0.0 in eng/SourceBuildPrebuiltBaseline.xml in that PR, and arcade brings 8.0.1. Is that okay to update those exclusions, or that should not have been done in the first place?

Those should not have been added as exclusions.

**Comment by nohwnd** [comment]
Ok, I will update the exclusions to unblock our migration, and investigate how to get rid of the exclusions right after: https://github.com/microsoft/vstest/issues/4927
